README.md

```
@@ -5,19 +5,24 @@ This repo builds the images for and runs in a podman pod the following services:
 
-- **Red Hat Developer Subscription** (required for building container images)
+- the `ocm` command (get it [here](https://console.redhat.com/openshift/token))
+
+- Either Vertex AI service account credentials or a Gemini API key. If using a Gemini API key, run `make generate` for instructions on how to get one.
+
+- An OpenShift pull-secret (https://cloud.redhat.com/openshift/install/pull-secret)
+    - Apply it to podman by copying it to ~/.config/containers/auth.json
+
+- Some report needing a **Red Hat Developer Subscription** (required for building container images)
+  - You might not need one
   - Sign up for free at https://developers.redhat.com/register
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)